libcxx/include/__locale_dir/support/apple.h

```diff
@@ -7,17 +6,15 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef _LIBCPP___LOCALE_DIR_LOCALE_BASE_API_APPLE_H
-#define _LIBCPP___LOCALE_DIR_LOCALE_BASE_API_APPLE_H
+#ifndef _LIBCPP___LOCALE_DIR_SUPPORT_APPLE_H
```

Comment:
Why are you introducing a new directory here?

Reply:
I want to move towards the following organization, which is what we do for the threading support:

```
__locale/
    support.h
    support/
        apple.h
        freebsd.h
        windows.h
        etc..
```

Reply:
So basically, I'm writing the "new" code under that organization and eventually the old stuff can be removed since it will be unused.